user/installing-dependencies.md

```diff
@@ -12,14 +12,18 @@ redirect_from:
 
 To install Ubuntu packages that are not included in the default [standard](/user/reference/precise/), use apt-get in the `before_install` step of your `.travis.yml`:
```

Comment:
I know this isn't related to your PR, but this would be better as:

> To install Ubuntu packages that are not included in the standard [precise](/user/reference/precise/) or [trusty](/user/reference/trusty/) distribution, use apt-get in the `before_install` step of your `.travis.yml`: